char/liberate/DATASHEET/gf180_12T_TT_3P3_25C.ccs/AND2_12T.html

```diff
@@ -0,0 +1,255 @@
+<HTML><BODY TEXT="#000000"  LINK="#0000FF" VLINK="#0000FF" ALINK="#00FFFF"><TABLE BORDER=0 CELLSPACING=0 CELLPADDING=5 WIDTH=100%>
+<TR align=center><th align=left><H1><font size=+3>AND2_12T</font></H1></th>
```

Comment:
are those generated?

Reply:
Yes, they are generated by the characterization

Reply:
if they are generated we might not want to commit them to the repo (and instead build + deploy them somewhere: rtfd.io, github pages).

@mithro, should we reuse the same doc infra as the other gf repo for consistency?

Reply:
They come from the characterizer, so I think they are useful.  They have information regarding each cell. On our in-house version, we are doing the same kind of thing.  So, I think they are useful.  Plus, you cannot generate them easily without a license as we used a commercial characterization tool for this first run.

Reply:
oh, I see. I'll let @mithro arbitrate on the best way to include files in the repo that people can't reproduce.